src/components/sbb-map-container/sbb-map-container.scss

```diff
@@ -0,0 +1,71 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
```

Comment:
Does the map somehow overflow page width? (see it extends purple line which marks page end) https://www.chromatic.com/test?appId=5ea7d9e48c451f00223b52ff&id=6425a4dec387b7f600979f82

Reply:
@jeripeierSBB As far as I can see there is nothing overflowing. Could this be an issue with chromatic?

Reply:
Could be, I will check it.

Reply:
It's probably because the width is 100vw. So maybe the question is, is it really the responsibility of the map container to ensure its stretching or should it be in responsibility of the consumer?